collection-scripts/gather_virtualization

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source $(dirname "$0")/common.sh
+BASE_COLLECTION_PATH="must-gather"
+get_operator_ns "kubevirt-hyperconverged"
+VIRTUALIZATION_PATH="${BASE_COLLECTION_PATH}/virtualization"
+mkdir -p "${VIRTUALIZATION_PATH}"
+
+function collect_running_vms_count() {
+  oc get vmi --all-namespaces -o=jsonpath='{.items[?(@.status.phase=="Running")].metadata.name}' | wc -w > "${VIRTUALIZATION_PATH}/running_vms_count.txt"
```

Comment:
must-gather is primarily for gathering diagnostics information; if you looking to send a metric back to Red Hat; you should use telemetry. 

In speaking with @djfjeff about this; the CNV team already has telemetry reporting for this and the purpose of this gather data (in this fashion) is to account for disconnected cluster too. With that being the case this data should be collected; via `registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9` 

However as I understand it the concern here (with including this collection in the cnv must-gather image) is this isn't always collected (by support or customers) so we don't have all the data we want (all the time). 

However with the work happening in https://issues.redhat.com/browse/CNV-29411 this data collection would duplicate what is already in the cnv must-gather image; so I am not in favor of including this (and am in favor of waiting for https://issues.redhat.com/browse/CNV-29411 to complete the offical solution).

Reply:
Note if https://github.com/kubevirt/must-gather/pull/191/ also merges; when a user collects both an OCP must-gather and a CNV must-gather you will spend time and data (size of the archive) collecting duplicate information. 

Its not a good ideal to implement collections in this manner.

Reply:
Hi, 

I understand that there could be duplication here, but this is a extremely important metric for us that we need to be collected at this time also from OCP for grater exposure. 
Once https://issues.redhat.com/browse/CNV-29411 will be implemented and we can relay on the SRE team to ask for the all the MGs, I promise we will remove this duplication.
I can add a story that will depend on https://issues.redhat.com/browse/CNV-29411 to remove it for tracking.

Reply:
@soltysh Please review.

Reply:
@sradco  I am not arguing the importance of the metric (to be clear; I am all for gathering all the metrics/data we can). What I am objecting to is the manner[s] in which the metric is being collected. 

Again; must-gather archives are a 'diagnostics tool'; a 'count' doesn't assist with diagnostics. It provides business insights. While we can (and do) gather 'business insights' from our diagnostics reports (to improve the product); these data points are extrapolated from the 'diagnostics information'. Ie: If you collect all the virtual machine instances (vmi's) object references as configurations a count can be extrapolated from that (for business insights purposes), but the data is also useful for 'diagnostics purposes'. 

Because this data is already collected: https://github.com/kubevirt/must-gather/blob/main/collection-scripts/gather_vms_details I don't see a strong justification to duplicate it. 

- Note: an argument to 'collect it in the primary must-gather image' is because the CNV image isn't always collected (and this requires additional back and forth); which can be true - again  https://issues.redhat.com/browse/CNV-29411 aims to address that problem (and given its focused on that problem it should be allowed to solve that problem (not this PR). 
    - Note: you can collect multiple archives at once with must-gather today; so the 'back and forth' issue while a valid usability issue; is manageable to day with training and education (on the SRE/Support/Customer side of the collection conversation). At least that is my opinion.